src/main/java/it/pn/frontend/e2e/pages/destinatario/personaGiuridica/PiattaformaNotifichePGPAPage.java

```diff
@@ -66,10 +66,10 @@ public void waitLoadPiattaformaNotificaPage(String ragioneSociale) {
             By codiceIunTextFieldBy = By.id("iunMatch");
             By dataInizioFieldBy = By.id("startDate");
             By dataFineFieldBy = By.id("endDate");
-            this.getWebDriverWait(this.loadComponentWaitTime).withMessage("Il titolo della pagina Notifiche PG non è visibile").until(ExpectedConditions.visibilityOfElementLocated(titlePageBy));
-            this.getWebDriverWait(40).withMessage("Il campo codice iun della pagina Notifiche PG non è cliccabile").until(ExpectedConditions.elementToBeClickable(codiceIunTextFieldBy));
-            this.getWebDriverWait(40).withMessage("Il campo data inizio della pagina Notifiche PG non è cliccabile").until(ExpectedConditions.elementToBeClickable(dataInizioFieldBy));
-            this.getWebDriverWait(40).withMessage("Il campo data fine della pagina Notifiche PG non è cliccabile").until(ExpectedConditions.elementToBeClickable(dataFineFieldBy));
+            getWebDriverWait(10).withMessage("Il titolo della pagina Notifiche PG non è visibile").until(ExpectedConditions.visibilityOfElementLocated(titlePageBy));
+            getWebDriverWait(10).withMessage("Il campo codice iun della pagina Notifiche PG non è cliccabile").until(ExpectedConditions.elementToBeClickable(codiceIunTextFieldBy));
+            getWebDriverWait(10).withMessage("Il campo data inizio della pagina Notifiche PG non è cliccabile").until(ExpectedConditions.elementToBeClickable(dataInizioFieldBy));
+            getWebDriverWait(10).withMessage("Il campo data fine della pagina Notifiche PG non è cliccabile").until(ExpectedConditions.elementToBeClickable(dataFineFieldBy));
```

Comment:
usare waitLoadComponentTime

Reply:
il metodo waitLoadComponentTime é utilizzato in basePage, é contenuto appunto in questo getWebDriverWait



![image](https://github.com/pagopa/pn-frontend-e2e/assets/119851973/6d8f4332-8e7e-4d3f-b3ea-641f63bc2940)
![image](https://github.com/pagopa/pn-frontend-e2e/assets/119851973/6d8f4332-8e7e-4d3f-b3ea-641f63bc2940)

Reply:
in questo caso (se ho ben capito) è inutile passare il parametro `long timeout` al metodo dato che quest'ultimo sarà sovrascritto, preferirei quindi un metodo senza parametri

Reply:
in questo caso ci serve, perché potremmo aver bisogno di un tempo maggiore per caricare un componente, in particolare su aws